internal/controllers/synthetic_devices_controller.go

```diff
@@ -2,14 +2,20 @@ package controllers
 
 import (
 	"context"
+	"math/big"
+	"math/rand"
 	"strconv"
+	"time"
 
 	"github.com/DIMO-Network/devices-api/internal/config"
+	registryContract "github.com/DIMO-Network/devices-api/internal/contracts"
```

Comment:
Can we take off this rename? It's long and looks like a variable name. Just see how it looks.